src/file_handler/file_loader.js

```diff
@@ -248,15 +246,16 @@ cwc.fileHandler.FileLoader.prototype.loadCWCFile = async function(data,
 
   // Handle tutorial data
   let tutorialInstance = this.helper.getInstance('tutorial');
-  if (tutorialInstance) {
-    tutorialInstance.setTutorial(file.getTutorial(userLanguage));
+  let tutorialSpec = file.getTutorial(userLanguage);
+  if (tutorialInstance && tutorialSpec) {
+    await tutorialInstance.setTutorial(tutorialSpec);
   }
 
   // Handle sidebar icons
   let sidebarInstance = this.helper.getInstance('sidebar');
   if (sidebarInstance) {
     sidebarInstance.enableDescription(file.getDescription());
-    sidebarInstance.enableTutorial(file.getTutorial(userLanguage));
+    sidebarInstance.enableTutorial(tutorialInstance.hasTutorial());
```

Comment:
Please revert, because this will cause an error message if the tutorialInstance ist not defined.